sql/testdata/builtin_function

```diff
@@ -332,3 +332,56 @@ SELECT extract(nanosecond from timestamp '2001-04-10 12:04:59.34565423')
 
 query error extract: unsupported timespan: nansecond
 SELECT extract(nansecond from timestamp '2001-04-10 12:04:59.34565423')
+
+query T
+SELECT abs(-1.0)
+----
+1
+
+query T
+SELECT abs(-0.0)
+----
+0
+
+query T
+SELECT abs(1.0)
+----
+1
+
+query error abs: abs of MinInt64 is not defined
+SELECT abs(-9223372036854775808)
```

Comment:
Might be good to test `-9223372036854775807` (the smallest integer for which `abs` is defined).